app/models/enumeration/EventType.scala

```diff
@@ -84,6 +84,7 @@ object EventType extends Enumerable.Implicits {
       case evType1827 if api1827Events.contains(evType1827) => ApiType.Api1827
       case evType1829 if api1829Events.contains(evType1829) => ApiType.Api1829
       case evType1830 if api1830Events.contains(evType1830) => ApiType.Api1830
+      case _ => throw new RuntimeException(s"API unknown for event type $eventType")
```

Comment:
Do we want to throw RuntimeExceptions in general? I think this is quite a broad net and maybe we could write in a custom exception for this.

Reply:
I think we should avoid throwing exceptions (side effects), instead we can return `Option[ApiType]`

Reply:
I have refactored code so that exception no longer needed. Uses options now instead.